Tella/Scenes/Settings/Views/Servers/AddServer/SuccessLoginView.swift

```diff
@@ -13,7 +28,8 @@ struct SuccessLoginView: View {
     
     @EnvironmentObject private var appViewState: AppViewState
     @State var showNextView : Bool = false
-    
+    var navigateToAction: () -> Void
+    var type: SuccessLoginType = .tella
```

Comment:
What do you think if we use `ServerConnectionType` instead of creating an other attribute (SuccessLoginType) ?

Reply:
and add `buttonContent` to  `ServerConnectionType` extension, maybe also update a significant name instead of `buttonContent`